react/index.tsx

```diff
@@ -83,11 +83,20 @@ if (window.ReactIntl) {
 if (window.__RUNTIME__.start && !window.__ERROR__) {
   if (canUseDOM) {
     const contentLoadedPromise = new Promise(resolve =>
-      document.addEventListener('DOMContentLoaded', resolve)
-    )
-    Promise.all([contentLoadedPromise, intlPolyfillPromise]).then(
-      window.__RENDER_8_RUNTIME__.start
+      window.addEventListener('DOMContentLoaded', resolve)
     )
+    Promise.all([contentLoadedPromise, intlPolyfillPromise]).then(() => {
+      setTimeout(() => {
+        performance.mark('render-start')
```

Comment:
is this preformance available in all browsers?

Reply:
👀

Reply:
It seems it does 
![image](https://user-images.githubusercontent.com/12702016/77168018-e00f6c80-6a95-11ea-8334-4d341b734834.png)